src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -101,7 +101,8 @@
     <PackageReference Include="System.Net.MsQuic.Transport"
                       Version="$(SystemNetMsQuicTransportVersion)"
                       PrivateAssets="all"
-                      GeneratePathProperty="true" />
+                      GeneratePathProperty="true" 
+                      Condition="'$(DotNetBuildFromSource)' != 'true'" />
```

Comment:
Should this same condition be applied to the target which consumes the file from this package?

Reply:
Nobody on the sourcebuild side consumes this, the installer packaging system picks up all the files in the net6.0-runtime folder and run some validations before creating the targeting packs.

Reply:
its used for interop calls mostly, no specific target uses this package